docs/setup/install-new.md

```diff
@@ -67,116 +67,158 @@ Example: ![Deployment Output](../images/deployment-output.png)
 
 ## Set up Application Gateway Ingress Controller
 
-### Overview
+With the instructions in the previous section we created and configured a new AKS cluster and
+an App Gateway. We are now ready to deploy an sample app and an ingress controller to our new
+Kubernetes infrastructure.
 
-With the instructions in the previous section we created and configured a new Azure Kubernetes Service (AKS) cluster. We are now ready to deploy to our new Kubernetes infrastructure. The instructions below will guide us through the proccess of installing the following 2 components on our new AKS:
+### Prerequisite: Kubernetes Credentials
+For the following steps we need setup [kubectl](https://kubectl.docs.kubernetes.io/) command,
+which we will use to connect to our new Kubernetes cluster. [Cloud Shell](https://shell.azure.com/) has `kubectl` already installed. We will use `az` CLI to obtain credentials for Kubernetes.
 
-1. **Azure Active Directory Pod Identity** - Provides token-based access to the [Azure Resource Manager (ARM)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview) via user-assigned identity. Adding this system will result in the installation of the following within your AKS cluster:
-   1. Custom Kubernetes resource definitions: `AzureIdentity`, `AzureAssignedIdentity`, `AzureIdentityBinding`
+Get credentials for your newly deployed AKS ([read more](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#connect-to-the-cluster)):
+    ```bash
+    az aks get-credentials --resource-group <your-new-resource-group> --name <name-of-new-AKS-cluster>
+    ```
+
+### Install AAD Pod Identity
+ Azure Active Directory Pod Identity provides token-based access to
+ [Azure Resource Manager (ARM)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview).
+ 
+ AAD Pod Identity will add the following components to your Kubernetes cluster:
+   1. Kubernetes [CRDs](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/): `AzureIdentity`, `AzureAssignedIdentity`, `AzureIdentityBinding`
    1. [Managed Identity Controller (MIC)](https://github.com/Azure/aad-pod-identity#managed-identity-controllermic) component
    1. [Node Managed Identity (NMI)](https://github.com/Azure/aad-pod-identity#node-managed-identitynmi) component
-1. **Application Gateway Ingress Controller** - This is the controller which monitors ingress-related events and actively keeps your Azure Application Gateway installation in sync with the changes within the AKS cluster.
 
-Steps:
 
-1. Configure `kubectl` with access to your newly deployed AKS:
+### Install App Gateway Ingress Controller
+Ingress Controller will monitor ingress-related events and will keep Azure Application Gateway
+in sync with the changes within the AKS cluster.
+
+1. Add [aad-pod-identity](https://github.com/Azure/aad-pod-identity) service to your cluster:
```

Comment:
Can you move add AAD Pod Identity service to the previous section ?

Reply:
I created `Prerequisites` section and in it now we have `Setup Kubernetes Credentials` and also `Install AAD Pod Identity` -- is that what you meant?

Also simplified the sample app YAML (but it won't work until we apply this fix: https://github.com/Azure/application-gateway-kubernetes-ingress/pull/553)

Reply:
I mean can you move it to #### Install AAD Pod Identity

Reply:
Got it! Makes sense!